spec/turn_spec.rb

```diff
@@ -0,0 +1,102 @@
+require './lib/turn'
+require './lib/card'
+
+RSpec.describe Turn do
+  before(:each) do
+    @card = Card.new("What is the capital of Alaska?", "Juneau", :Geography)
+    @turn = Turn.new("Juneau", @card)
+  end
+
+  describe Card do
```

Comment:
Since the `Card` class was tested in `card_spec.rb`, this describe block might not be necessary here.